pkg/commands/commit.go

```diff
@@ -52,5 +53,12 @@ func (c *Commit) GetDisplayStrings(isFocused bool) []string {
 		actionString = cyan.Sprint(utils.WithPadding(c.Action, 7)) + " "
 	}
 
-	return []string{shaColor.Sprint(c.Sha), actionString + white.Sprint(c.Name)}
+	name := ""
```

Comment:
I think we should follow the same pattern as above and color the sha magenta rather than the commit message itself

Reply:
I fixed this :)